AUTHORS

```diff
@@ -31,6 +31,7 @@ Brian Quistorff
 Brian Van Essen
 captain123
 Carlos Silla
+cerrisantos
```

Comment:
It is strange that your commits include that change. Please try to remove it.

You can use `git reset upstream/master` and then re-create your commit using `git gui`.